static/select_seats.css

```diff
@@ -10,25 +10,38 @@ div.seat-info div {
 
 .theater-seat-map svg {
     max-width: 400px;
-    max-height: 400px;
+    max-height: 480px;
+    viewBox: "0 0 400 400"
```

Comment:
I'm pretty sure you can't set the viewbox in CSS, but if you could it would be in kebabcase `view-box`.